admin/tools/docker/integration-tests/localhost/env.example.sh

```diff
@@ -0,0 +1,21 @@
+# VERSION can be ticket branch but with _ instead of /
```

Comment:
This file was added to the repo without execute permissions -- was that intentional.  Ie., it should only ever be sourced?